### PR TITLE
Should be able to specify kubernetes node labels per node pool (#59)

### PR DESCRIPTION
**Pull request by blackdog0403**
User can specify kubernetes labels per **node pool.**
- **nodepool -> user can ensure a nodepool have one more label**

Can apploy Array of labels to kubernetes nodes in config.yaml

- defaultAwsMasterNode 
- defaultAwsClusterNode
- defaultAwsSpecialNode

Default Label - nodepool={{ node.name }} - is left.

### CODE REVIEW

**Comment by blackdog0403** [comment]
@joejulian Do we have to update schema for non breaking change? (I added optional property for node configuration)

**Comment by joejulian** [comment]
Yes, non-breaking changes still go in but do not require a version update.

File: Documentation/kraken-configs/node/aws-node.md [hunk 1]
```diff
@@ -19,7 +19,8 @@
 | provider       | __Required__ | String       | provider     |
 | type           | __Required__ | String       | type of node to be launched - will vary depending on provider |
 | subnet         | __Required__ | String Array | Describes AWS subnets to be created per AZ |
-| tags           |   Optional   | Object Array | Array of tags to apply to node. |
+| labels           |   Optional   | Object Array | Array of tags to apply to node. |
+| tags           |   Optional   | Object Array | Array of labels to apply to kubernetes nodes. ( defaultAwsMasterNode, defaultAwsClusterNode, defaultAwsSpecialNode) |
```
Comment (coffeepac) [inline Documentation/kraken-configs/node/aws-node.md]: I think these are backwards.  ie:  labels are applied to node objects in kubernetes and and tags are applied to the physical node.

**Comment by coffeepac** [comment]
ok to test

**Comment by blackdog0403** [comment]
@coffeepac I'm working on write schema and will check document again.

File: Documentation/kraken-configs/node/aws-node.md (hunk 1 above)
Comment (blackdog0403) [inline Documentation/kraken-configs/node/aws-node.md]: Fixed it!

**Comment by blackdog0403** [comment]
@Minkyu-Choi test

**Comment by blackdog0403** [comment]
@joejulian Please check again~

**Comment by joejulian** [comment]
retest this please

**Comment by davidewatson** [comment]
awsKubeNodeLabel.json might be better named kubeNodeLabel, or similar, since this kubernetes feature is not provider specific...

**Pull request merged**